app/assets/javascripts/common/password_validation.js

```diff
@@ -0,0 +1,53 @@
+PasswordValidation = function() {
+  const DebounceTimeout = 500;
+
+  this.initialize = function() {
+    this.timer = null;
+    this.response = null;
+    this.passwordInput = $("#password");
+    this.passwordInput.on("input", this.handlePasswordInput);
+  }
+
+  this.handlePasswordInput = () => {
+    console.log("CHANGE", this.passwordInput.val());
+    this.setTimer();
+  }
+
+  this.setTimer = function() {
+    console.log("STARTING TIMER")
+    this.cancelTimer();
```

Comment:
Some feedback for the follow up PR:

I would suggest using the same [debounce pattern](https://github.com/simpledotorg/simple-server/blob/e906b5efcfa7d28945fcd28f5e593efb149b3e58/app/assets/javascripts/common/regions_search.js#L73-L86) we have elsewhere in the app, I think its more clear and easier to follow than this.

Also, you can use ECMAscript's `=>` and the like to clean up some of the line noise in here.